src/Interactive/VbiCore/vbi.rsp

```diff
@@ -0,0 +1,35 @@
+/r:System.Collections.dll
```

Comment:
Are there any VB-specific references we should add?  What's in the default response file for vbc?